app/views/posts/show.html.erb

```diff
@@ -7,11 +7,22 @@
             <span>comments: <%= @post.comments.count %>,</span>
             <span>likes: <%= @post.likes.count %></span>
         </div>
-        <p><%= @post.text %></p>
+        <p><%= @post.title %></p>
        
         <% @post.comments.each do |comment| %>
             <span class='commentor'><%= comment.author.name %>: </span>
             <span><%= comment.text %></span><br>
         <% end %>
     </div>
-    <% index = index + 1 %>
\ No newline at end of file
+<% index = index + 1 %>
+
+<%= form_with(model: @comment, url: "/users/#{@post.author.id}/posts/#{@post.id}/comments", method: :post, scope: :comment) do |form| %>
+  <div class="form-group">
+    <%= form.label :text %>
+    <%= form.text_area :text, class: 'form-control' %>
+  </div>
+
+  <%= form.submit 'Create Comment', class: 'btn btn-primary' %>
```

Comment:
- [ ] As suggested by the previous reviewer that you should have this form in **new.html.erb** file because when you make will make a new method in the comments controller which I mentioned in the previous inline comment then you will need view and **new.html.erb** is the view which you need :+1: